### PR TITLE
Exclude `.ipynb_checkpoints` by default

### PR DESCRIPTION
**Pull request by chrisjsewell**
Idea came from spatialaudio/nbsphinx#440

### CODE REVIEW

**Comment by choldgraf** [comment]
Nice 👍

Hmm I'm not sure where that admonition warning is coming from.

**Comment by chrisjsewell** [comment]
> Is it related to myst parser?

In 792beaf I tryed pinnin myst-parser to the [penultimate version](https://github.com/ExecutableBookProject/MyST-Parser/releases), and still getting the error.

Weird/annoying that it doesn't give a source location?

**Comment by choldgraf** [comment]
Hmm, I wonder if it could be related to the theme somehow? Do we get the error if we switch to the rtd theme?

**Comment by chrisjsewell** [comment]
Also in c3381f5 I commented out the new code, and it doesn't look related to that. So yeh surely something to do with a dependant version change somewhere

**Comment by chrisjsewell** [comment]
From this build:

```
Successfully installed GitPython-3.1.1 Jinja2-2.11.2 MarkupSafe-1.1.1 Pygments-2.6.1 Send2Trash-1.5.0 alabaster-0.7.12 altair-4.1.0 async-generator-1.10 babel-2.8.0 backcall-0.1.0 bleach-3.1.4 bokeh-2.0.1 colorama-0.4.3 cycler-0.10.0 decorator-4.4.2 defusedxml-0.6.0 docutils-0.16 entrypoints-0.3 gitdb-4.0.4 imagesize-1.2.0 ipykernel-5.2.1 ipython-7.13.0 ipython-genutils-0.2.0 ipywidgets-7.5.1 jedi-0.17.0 jupyter-cache-0.2.1 jupyter-client-6.1.3 jupyter-core-4.6.3 jupyter-sphinx-0.2.4a1 kiwisolver-1.2.0 latexcodec-2.0.0 markdown-it-py-0.4.5 matplotlib-3.2.1 mistune-0.8.4 mpmath-1.1.0 myst-nb-0.7.0 myst-parser-0.8.1 nbclient-0.2.0 nbconvert-5.6.1 nbdime-2.0.0 nbformat-5.0.6 nest-asyncio-1.3.2 notebook-6.0.3 numpy-1.18.3 oset-0.1.3 packaging-20.3 pandas-1.0.3 pandocfilters-1.4.2 parso-0.7.0 pickleshare-0.7.5 pillow-7.1.1 plotly-4.6.0 prometheus-client-0.7.1 prompt-toolkit-3.0.5 pybtex-0.22.2 pybtex-docutils-0.2.2 pydata-sphinx-theme-0.2.0 python-dateutil-2.8.1 pytz-2019.3 pyyaml-5.3.1 pyzmq-19.0.0 retrying-1.3.3 smmap-3.0.2 snowballstemmer-2.0.0 sphinx-2.4.4 sphinx-copybutton-0.2.10 sphinx-togglebutton-0.1.0 sphinxcontrib-applehelp-1.0.2 sphinxcontrib-bibtex-1.0.0 sphinxcontrib-devhelp-1.0.2 sphinxcontrib-htmlhelp-1.0.3 sphinxcontrib-jsmath-1.0.1 sphinxcontrib-qthelp-1.0.3 sphinxcontrib-serializinghtml-1.1.4 sqlalchemy-1.3.16 sympy-1.5.1 terminado-0.8.3 testpath-0.4.4 toolz-0.10.0 tornado-6.0.4 traitlets-4.3.3 typing-extensions-3.7.4.2 wcwidth-0.1.9 widgetsnbextension-3.5.1
```

From last successful build:

```
Successfully installed GitPython-3.1.1 Jinja2-2.11.2 MarkupSafe-1.1.1 Pygments-2.6.1 Send2Trash-1.5.0 alabaster-0.7.12 altair-4.1.0 async-generator-1.10 babel-2.8.0 backcall-0.1.0 bleach-3.1.4 bokeh-2.0.1 colorama-0.4.3 cycler-0.10.0 decorator-4.4.2 defusedxml-0.6.0 docutils-0.16 entrypoints-0.3 gitdb-4.0.4 imagesize-1.2.0 ipykernel-5.2.1 ipython-7.13.0 ipython-genutils-0.2.0 ipywidgets-7.5.1 jedi-0.17.0 jupyter-cache-0.2.1 jupyter-client-6.1.3 jupyter-core-4.6.3 jupyter-sphinx-0.2.4a1 kiwisolver-1.2.0 latexcodec-2.0.0 markdown-it-py-0.4.5 matplotlib-3.2.1 mistune-0.8.4 mpmath-1.1.0 myst-nb-0.7.0 myst-parser-0.8.1 nbclient-0.2.0 nbconvert-5.6.1 nbdime-2.0.0 nbformat-5.0.6 nest-asyncio-1.3.2 notebook-6.0.3 numpy-1.18.3 oset-0.1.3 packaging-20.3 pandas-1.0.3 pandocfilters-1.4.2 parso-0.7.0 pickleshare-0.7.5 pillow-7.1.1 plotly-4.6.0 prometheus-client-0.7.1 prompt-toolkit-3.0.5 pybtex-0.22.2 pybtex-docutils-0.2.2 pydata-sphinx-theme-0.1.1 python-dateutil-2.8.1 pytz-2019.3 pyyaml-5.3.1 pyzmq-19.0.0 retrying-1.3.3 smmap-3.0.2 snowballstemmer-2.0.0 sphinx-2.4.4 sphinx-copybutton-0.2.10 sphinx-togglebutton-0.1.0 sphinxcontrib-applehelp-1.0.2 sphinxcontrib-bibtex-1.0.0 sphinxcontrib-devhelp-1.0.2 sphinxcontrib-htmlhelp-1.0.3 sphinxcontrib-jsmath-1.0.1 sphinxcontrib-qthelp-1.0.3 sphinxcontrib-serializinghtml-1.1.4 sqlalchemy-1.3.16 sympy-1.5.1 terminado-0.8.3 testpath-0.4.4 toolz-0.10.0 tornado-6.0.4 traitlets-4.3.3 typing-extensions-3.7.4.2 wcwidth-0.1.9 widgetsnbextension-3.5.1
```

Spot the difference! 👀

**Comment by choldgraf** [comment]
good to know haha, ok will try to investigate in the theme then

**Comment by choldgraf** [comment]
definitely here: https://github.com/pandas-dev/pydata-sphinx-theme/pull/123/files#diff-f4ceb49a953833677364ee51c8bb00fbR56

File: setup.py [hunk 1]
```diff
@@ -77,7 +77,7 @@
             "plotly",
             "matplotlib",
             "sphinx-copybutton",
-            "pydata-sphinx-theme~=0.1",
+            "pydata-sphinx-theme==0.1.1",
```
Comment (choldgraf) [inline setup.py:80]: 😭😭😭😭😭😭

Comment (chrisjsewell) [reply setup.py:80]: Yep that's fixed it.

Weird that this didn't work:

```
Collecting pydata-sphinx-theme~=0.1
  Downloading pydata_sphinx_theme-0.2.0-py2.py3-none-any.whl (38 kB)
```

Comment (choldgraf) [reply setup.py:80]: next release will be v0.2.1 anyway, since I'll be the one that does it tomorrow probably haha

Comment (chrisjsewell) [reply setup.py:80]: yep, I'll wait for that to merge 👍

File: setup.py [hunk 2]
```diff
@@ -77,7 +77,7 @@
             "plotly",
             "matplotlib",
             "sphinx-copybutton",
-            "pydata-sphinx-theme",
+            "pydata-sphinx-theme==0.1.1",
```
Comment (choldgraf) [inline setup.py:80]:
```suggestion
            "pydata-sphinx-theme",
```

**Comment by choldgraf** [comment]
OK I just made a minor release, let's see if the bug is fixed

**Pull request merged**